src/components/permissions/permission-chip-selector.tsx

```diff
@@ -37,12 +40,30 @@ export class PermissionChipSelector extends React.Component<IProps, IState> {
       return (
         <LabelGroup>
           {items.map((text) => (
-            <Label key={text}>{text}</Label>
+            <Label key={text}>
+              {this.props.multilingual ? i18n._(text) : text}
+            </Label>
           ))}
         </LabelGroup>
       );
     }
 
+    let selections = [];
+    if (this.props.multilingual) {
+      this.props.selectedPermissions.forEach((item) => {
+        const item2 = {};
+        // orginal english value
+        item2['value'] = item;
+        item2.toString = function (): string {
+          // translated value
+          return i18n._(item);
+        };
+        selections.push(item2);
+      });
```

Comment:
Tsk, this sounds like a use for `map` ;)

```suggestion
      selections = this.props.selectedPermissions.map((value) => ({
        // orginal english value
        value,
        // translated
        toString: () => i18n._(value),
      }));
```

(+- comments)